pkgs/development/python-modules/drf-pydantic/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  django,
+  pydantic,
+  hatchling,
+  djangorestframework,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "drf-pydantic";
+  version = "v2.7.1";
```

```suggestion
  version = "2.7.1";
```

The package version does not include any prefixes like that. That is for the tag only.